style.css

```diff
@@ -1,112 +1,97 @@
-
```

Comment:
- [ ] It would be so much better if you follow one of the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). Please choose one and follow the design.  💪 👍